ui/src/utils/validateTaskPayloadSchemas.js

```diff
@@ -0,0 +1,66 @@
+import ajv from './ajv';
+import urls from './urls';
+
+const schemas = {};
+const fetchSchema = async (service, schema) => {
+  const res = await fetch(urls.schema(service, schema));
+
+  return res.json();
+};
+
+const prefetch = async () => {
+  ajv.addSchema(await fetchSchema('common', 'metaschema.json'));
+  ajv.addSchema(await fetchSchema('queue', 'v1/task.json'));
+  ajv.addSchema(await fetchSchema('queue', 'v1/task-metadata.json'));
+};
+
+const prefetchPromise = prefetch();
+
+export const formatErrorDetails = error => {
+  const msg = [error.message];
+
+  if (error.keyword === 'type') {
```

Comment:
other types are quite self-explanatory in message, no need to add extra messages